src/typings/index.d.ts

```diff
@@ -98,6 +98,23 @@ interface DropdownProps {
   sideHeading?: string
 }
 
+interface ButtonDropdownProps {
+  handleChange: (value: SelectValue) => void
+  value: SelectValue
```

Comment:
this should be `value?: SelectValue | null`

Reply:
Done